config/locales/en/documents/history.yml

```diff
@@ -24,8 +25,11 @@ en:
         draft_reset: "Draft reset to last published edition"
         unwithdrawn: "Unwithdrawn"
         scheduled: "Scheduled to publish"
+        scheduled_publishing_succeeded: "Published on schedule"
+        scheduled_publishing_without_review_succeeded: "Published on schedule but needs 2i review"
```

Comment:
Minor, but I notice the 'needs 2i review' is inconsistent compared to 'without review' above. Is that intentional?

Reply:
I felt that it was more important to have the key consistent with `publishing_without_review` than it was to be consistent with the copy of the text. I want to check in with @benthorner about the text copy as a step of completing this feature.